ext/pdo_pgsql/config.m4

```diff
@@ -71,11 +71,8 @@ if test "$PHP_PDO_PGSQL" != "no"; then
   grep openssl $PGSQL_INCLUDE/libpq-fe.h >/dev/null 2>&1
   if test $? -eq 0 ; then
     AC_MSG_RESULT([yes])
-    dnl First try to find pkg-config
-    AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
-    if test -x "$PKG_CONFIG" && $PKG_CONFIG --exists openssl; then
-      PDO_PGSQL_CFLAGS=`$PKG_CONFIG openssl --cflags`
-    fi
+    PKG_CHECK_MODULES([PDO_PGSQL_SSL], [openssl], ,
+      [AC_MSG_WARN("openssl CFLAGS not found")])
   else
```

Comment:
Was it intentional that this used to just silently skip over everything if it couldn't find openssl.pc?

Reply:
The whole thing looks super fishy. It greps `libpq-fe.h` for `openssl`, but looking at the file (https://github.com/postgres/postgres/blob/master/src/interfaces/libpq/libpq-fe.h) it only contains `OpenSSL`. I'm not even sure what exactly this is supposed to be checking and whether it's actually necessary.

Reply:
This was introduced in https://github.com/php/php-src/commit/0db373883fb073e1773823b236daaf0d3f5a0491. Given the commit message and the fact that was done in 2005, I'd assume that this is no longer relevant and we can just drop this code entirely.

Reply:
It used to:

```
#ifdef USE_SSL
#include <openssl/ssl.h>
#endif
```

Ironically, not too long after the patch to php, postgres removed it in https://github.com/postgres/postgres/commit/3c4768d0d17d4569f2417aa5741e0317404b6c45

So yeah, it did nothing.